requirements/tests.txt

```diff
@@ -1,3 +1,4 @@
+dask
```

Comment:
`dask[array]` I believe?

Reply:
I don't think we are using anything from `dask.array`. The only imports are from `dask.base`.

Reply:
Oops, I was completely wrong about that. Indeed, `dask[array]` is what we needed. Pushed new commit. :-D